cpp/test/Ice/invoke/AllTests.cpp

```diff
@@ -646,6 +703,8 @@ allTests(Test::TestHelper* helper)
         {
             test(false);
         }
+        test(result);
+        test(outEncaps.size() == 0 || outEncaps.size() == 6); // Empty encapsulation
```

Comment:
Should be 0, not 6. We know how it's implemented locally - doesn't depend on the peer.